common/lib/container-definitions/package-lock.json

```diff
@@ -569,8 +569,8 @@
       }
     },
     "@fluidframework/protocol-definitions": {
-      "version": "0.1025.1",
-      "resolved": "https://registry.npmjs.org/@fluidframework/protocol-definitions/-/protocol-definitions-0.1025.1.tgz",
+      "version": "0.1026.0-0",
+      "resolved": "https://registry.npmjs.org/@fluidframework/protocol-definitions/-/protocol-definitions-0.1026.0-0.tgz",
```

Comment:
this looks wrong. this should point to the concrete version. my guess is that this has something to do with symlink

Reply:
if i do the same change locally i see this in my package lock:
``` json
        "@fluidframework/protocol-definitions": {
          "version": "0.1025.1",
          "resolved": "https://registry.npmjs.org/@fluidframework/protocol-definitions/-/protocol-definitions-0.1025.1.tgz",
          "integrity": "sha512-xlMWlDb+5xYzxEN2Z2SxOy0a8Im7IOWhBEYfaXCroHUrjn6yXw2BbVqGK4D97loyBdLhWYAqUnIErwJEciezPw==",
          "dev": true,
          "requires": {
            "@fluidframework/common-definitions": "^0.20.0"
          }
```

Reply:
Should this change be undone, then? Or should it be different (as in, "1026.0-42695")?

Reply:
maybe you did find replace in you package lock. that won't work. you need to actually run npm install